DontForget/app/src/main/java/com/test/dontforget/Repository/TodoRepository.kt

```diff
@@ -51,6 +51,7 @@ class TodoRepository {
                         a1.ref.child("todoMemo")
                             .setValue(todoDataClass.todoMemo)
                             .addOnCompleteListener(callback1)
+                        a1.ref.child("todoOwnerName").setValue(todoDataClass.todoOwnerName)
```

Comment:
이 부분 53번째 줄  `.addOnCompleteListener(callback1)` 을 54번째 줄 `a1.ref.child("todoOwnerName").setValue(todoDataClass.todoOwnerName)` 밑으로 넣어두면 좋을 것 같습니다.

```suggestion
                        a1.ref.child("todoMemo").setValue(todoDataClass.todoMemo)
                        a1.ref.child("todoOwnerName")
                            .setValue(todoDataClass.todoOwnerName)
                            .addOnCompleteListener(callback1)
```